ncm-spma/src/main/pan/components/spma/yumng/config.pan

```diff
@@ -0,0 +1,22 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+unique template components/spma/yumng/config;
+
+# Prefix for packages/groups
+prefix '/software';
+'groups/yumng/names' ?= list();
```

Comment:
This should be 'groups/names' ?= list(); but even better, let's move it to schema.pan